resources/i18n/it_IT/cura.po

```diff
@@ -1977,7 +1810,7 @@ msgstr "Nascondi tutte le stampanti collegate"
 
 msgctxt "@action:button"
 msgid "Hide settings"
-msgstr ""
+msgstr "Nascondere le impostazioni"
```

Comment:
It should be be translated to "Nascondi le impostazioni" because it does the action to hide...and "to hide" is "nascondere"

Reply:
I definitely trust you :smiley: